administrator/language/de-DE/com_joomlaupdate.ini

```
@@ -74,38 +93,39 @@ COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_NONE="Es sind keine Erweiterungen insta
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_PRE_UPDATE_CHECKS_FAILED="Kompatibilitätsprüfung fehlgeschlagen"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_PRE_UPDATE_CHECKS_FAILED_NOTES="Es war nicht möglich, die Kompatibilität aller Erweiterungen zu prüfen. Die Abfrage beim Update-Server hat entweder ein Zeitlimit überschritten oder einen Fehler zurückgegeben."
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_PROBABLY_COMPATIBLE="Kein Update erforderlich"
-COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_PROBABLY_COMPATIBLE_NOTES="<p>Der Entwickler der Erweiterung gibt an, dass die derzeit installierte Version kompatibel ist.</p><p id='updateyellowwarning' class='hidden'>Bitte beachten, ist eine Version mit <span class='label label-warning'>X.X.X</span> gekennzeichnet, wird vom Entwickler für die aktuelle Version von Joomla eine neuere Version der Erweiterung angeboten, als für die neue Version von Joomla. Bitte vor dem Update den Entwickler kontaktieren.</p>"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_PROBABLY_COMPATIBLE_NOTES="<p>Der Entwickler der Erweiterung gibt an, dass die derzeit installierte Version kompatibel ist.</p><p id=\"updateyellowwarning\" class=\"hidden\">Bitte beachten, ist eine Version mit <span class='label label-warning'>X.X.X</span> gekennzeichnet, wird vom Entwickler für die aktuelle Version von Joomla eine neuere Version der Erweiterung angeboten, als für die neue Version von Joomla. Bitte vor dem Update den Entwickler kontaktieren.</p>"
 COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_REQUIRING_UPDATES_TO_BE_COMPATIBLE="Update erforderlich"
-COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_REQUIRING_UPDATES_TO_BE_COMPATIBLE_NOTES="<p>Bitte die Erweiterungen aktualisieren, bevor Joomla aktualisiert wird.</p><p id='updateorangewarning' class='hidden'>Bitte besonders vorsichtig sein, wenn die zu aktualisierende Version der Erweiterung nicht auch als kompatibel mit der aktuellen Version von Joomla aufgeführt ist.</p>"
+COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_REQUIRING_UPDATES_TO_BE_COMPATIBLE_NOTES="<p>Bitte die Erweiterungen aktualisieren, bevor Joomla aktualisiert wird.</p><p id=\"updateorangewarning\" class=\"hidden\">Bitte besonders vorsichtig sein, wenn die Erweiterung auch als inkompatibel mit der aktuellen Version von Joomla gelistet ist.</p>"
```

```suggestion
COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_REQUIRING_UPDATES_TO_BE_COMPATIBLE_NOTES="<p>Bitte die Erweiterungen aktualisieren, bevor Joomla aktualisiert wird.</p><p id=\"updateorangewarning\" class=\"hidden\">Bitte besonders vorsichtig sein, wenn die Erweiterung auch als inkompatibel mit der aktuellen Version von Joomla! gelistet ist.</p>"
```

---

Wenn du jetzt das "!" verwenden willst - da ist noch ein Zweites im Satz:
```suggestion
COM_JOOMLAUPDATE_VIEW_DEFAULT_EXTENSIONS_REQUIRING_UPDATES_TO_BE_COMPATIBLE_NOTES="<p>Bitte die Erweiterungen aktualisieren, bevor Joomla! aktualisiert wird.</p><p id=\"updateorangewarning\" class=\"hidden\">Bitte besonders vorsichtig sein, wenn die Erweiterung auch als inkompatibel mit der aktuellen Version von Joomla! gelistet ist.</p>"
```

---

Erste wäre für mich echt nicht nötig.

---

Da Die Richtlinie sowieso nicht streng durchgesetzt wird, spielt das keine Rolle - nimm deine Version